src/components/test/NoticeBox.test.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * @jest-environment jsdom
+ */
+/* global expect */
+
+/**
+ * External dependencies
+ */
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+
+/**
+ * Internal dependencies
+ */
+import { NoticeBox } from "../NoticeBox";
+
+describe("with notices to display", () => {
+  describe("without a hasBeenIndexed date", () => {
+    test("not-indexed notice is displayed", () => {
+      const { rerender } = render(
+        <NoticeBox
+          recordCount={20}
+          planRecordLimit={100}
+          hasBeenIndexed={false}
+          hasValidData={true}
+          hasItems={true}
+        ></NoticeBox>
+      );
+
+      expect(
+        screen.getByText("Your content has not yet been indexed for Search")
+      ).toBeVisible();
+    });
+  });
+
+  describe("without valid data", () => {
```

Comment:
Hi @annacmc Sorry I didn't say it clear enough. I meant you could wrap a group of related tests into a describe, like the following. 

```JavaScript
  describe("A functionality to test", () => {
    // a test 
    test("should display correct", () => {
      const { rerender } = render(
        <NoticeBox
          recordCount={20}
          planRecordLimit={100}
          hasBeenIndexed={true}
          hasValidData={false}
          hasItems={true}
        ></NoticeBox>
      );

      expect(
        screen.getByText(
          "We weren’t able to properly locate your content for Search"
        )
      ).toBeVisible();
    });
   // another test
    test("unable to access data notice is displayed", () => {
      const { rerender } = render(
        <NoticeBox
          recordCount={20}
          planRecordLimit={100}
          hasBeenIndexed={true}
          hasValidData={false}
          hasItems={true}
        ></NoticeBox>
      );

      expect(
        screen.getByText(
          "We weren’t able to properly locate your content for Search"
        )
      ).toBeVisible();
    });
  });
```

If there is only one `test` inside a `describe`, then I guess using only `test` would be fine. But either ways, this would throw proper failure notice when things aren't right 😄

Reply:
Yeah, that makes a lot of sense! I started out that way.. then just kept going 😂  I think I went a little overboard on the describes 🙈  you're write they do through proper failure notices now, but I think I could scale back some of the describes 😆